troubleshooting/debugging.md

```diff
@@ -169,6 +169,14 @@ aws ec2 modify-instance-metadata-options \
      --http-endpoint enabled
 ```
 
+The other way to override the IMDS issue, without making the hop-limit to 2 will be setting up the hostNetwork to true, with the dnsPolicy as 'ClusterFirstWithHostNet':
+
+```
+    spec:
+      dnsPolicy: ClusterFirstWithHostNet
+      hostNetwork: true
+```
+
 #### Searching old issues
```

Comment:
@MohanRajRavi Thanks- can you add something here to make it very clear that this applies to EKS/k8s customers.  Since the above also works for ECS, and some folks might get confused.

Reply:
Thanks @PettitWesley updated the doc.